proseco/guide.py

```diff
@@ -183,19 +183,34 @@ def search_stage(self, stage):
 
         n_sigma = stage['SigErrMultiplier']
 
+        # For the reasonable mag check, override the SigErrMultiplier to be at least 1.
+        # It may go to 0 in later stages for the spoiler checks.
+        mag_err_mult = max(n_sigma, 1)
+
+        # And for bright stars, use a local mag_err that is lower bounded at 0.1
+        # for the mag selection.
+        mag_err = cand_guides['mag_err']
+        bright = cand_guides['mag'] < 7.0
+        mag_err[bright] = mag_err[bright].clip(0.1)
```

Comment:
I also figure that there's really no big issue in just bounding the effective mag_err itself for this specific check.  So when combined with the stages and the SigErrMultipliers, this is effectively:

Stage 1: limit 6.2, multiplier 3, brightest possible star 6.5
Stage 2: limit 6.2, multiplier 2, brightest possible star 6.4
Stage 3: limit 6.1, multiplier 1, brightest possible star 6.2
Stage 4: limit 6.0, multiplier 1, brightest possible star 6.1
Stage 5: limit 5.9, multiplier 1, brightest possible star 6.0